mjcf_to_sdformat/mjcf_to_sdformat/mjcf2sdformat.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from dm_control import mjcf
+
+from mjcf_to_sdformat.converters.model import add_mjcf_worldbody_to_sdf
+
+import sdformat as sdf
+
+
+def mjcf_file_to_sdformat(model_file, output_file):
+    mjcf_model = mjcf.from_path(model_file)
+    root = sdf.Root()
+    world = sdf.World()
+    world.set_name("default")
+
+    add_mjcf_worldbody_to_sdf(mjcf_model, world)
+
+    root.add_world(world)
+
+    f = open(output_file, "w")
```

Comment:
Consider using a context manager:
```suggestion
    with open(output_file, "w") as f:
      f.write(root.to_string())
```